apps/editing-toolkit/editing-toolkit-plugin/wpcom-block-editor-nux/src/tour-content.js

```diff
@@ -0,0 +1,83 @@
+/*** THIS MUST BE THE FIRST THING EVALUATED IN THIS SCRIPT *****/
+import './public-path';
+
+/**
+ * External dependencies
+ */
+import { __ } from '@wordpress/i18n';
+
+/**
+ * This function returns a collection of NUX Tour slide data
+ *
+ * @returns { Array } a collection of <WelcomeTourCard /> props
+ */
+function getTourContent() {
+	return [
+		{
+			heading: __( 'Learn the basics', 'full-site-editing' ),
+			description: __(
+				'Take the tour to learn the fundamentals of the WordPress editor.',
+				'full-site-editing'
+			),
+			imgSrc: 'https://nuxtourtest.files.wordpress.com/2020/11/mock-slide-0-2.jpg?resize=400px',
```

Comment:
How much extra work would it be to preload the GIFs? Even if we preloaded the next slide's GIF?

They have the potential to be larger than normal images, and it might be good to prevent jumping:

![Nov-24-2020 14-14-12](https://user-images.githubusercontent.com/6458278/100042500-ae8a4480-2e5f-11eb-8f87-41ba11c90eef.gif)

Reply:
Yes, there is definitely lag time for image load that needs to be addressed.  Added that to [Tech ToDos in MVP issue/ticket](https://marketplace.visualstudio.com/items?itemName=cssho.vscode-svgviewer)

Reply:
@roo2  See above discussion.  This might be helpful https://stackoverflow.com/questions/42615556/how-to-preload-images-in-react-js

Reply:
I have a simpler solution to this issue which is to set the height on the containing div. So the image will still take a sec to load but it won't jump all over the place ;)

Reply:
In case it helps, this is how it's been done for the design previews, https://github.com/Automattic/wp-calypso/blob/c663e6f4ff997c6a769dfae978f49ec9dcc54c8b/client/landing/gutenboarding/available-designs.ts#L60